article.tex

```diff
@@ -1,4 +1,4 @@
-\documentclass[pointlessnumbers]{scrreprt} %KOMA-Script Klasse für report; 
+\documentclass[pointlessnumbers]{scrartcl} %KOMA-Script Klasse für article; 
```

Comment:
I don't understand why you want to switch the document class. scrartcl doesn't have chapters.